addons/bestja_volunteer/wizards.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+from openerp import models, fields, api
+
+
+class Wizard(models.TransientModel):
+    _name = 'bestja.report_account_deletion_wizard'
+
+    reason = fields.Text(string="Dlaczego chcesz usunąć konto?", required=True)
```

Comment:
Why is this a required field? Can’t we delete account without giving any reason?